src/main/java/core/basesyntax/model/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import core.basesyntax.Machine;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Common Excavator, You can do it");
```

Comment:
The message 'Common Excavator, You can do it' is not very informative. Consider specifying what work the Excavator is starting, for example, 'Excavator starts digging'.